Extensions/better_reblogs.js

```diff
@@ -220,6 +220,16 @@ XKit.extensions.better_reblogs = new Object({
             if (!(this.preferences.margin.value || this.preferences.remove_user_names.value)) {
                 XKit.tools.add_css(".reblog-list-item .reblog-content {margin-left: 3px;}", "better_reblogs");
             }
+
```

Comment:
The standard should be tabs. It's odd that this file doesn't conform to what is specified in the [.editorconfig file](https://github.com/new-xkit/XKit/blob/master/.editorconfig).